tests/skipped_tests.tbl

```diff
@@ -60,18 +60,6 @@ tests/third_party/cupy/indexing_tests/test_generate.py::TestAxisConcatenator::te
 tests/third_party/cupy/indexing_tests/test_generate.py::TestC_::test_c_1
 tests/third_party/cupy/indexing_tests/test_generate.py::TestC_::test_c_2
 tests/third_party/cupy/indexing_tests/test_generate.py::TestC_::test_c_3
-tests/third_party/cupy/indexing_tests/test_generate.py::TestRavelMultiIndex::test_basic
```

Comment:
Can we add `tests/third_party/cupy/indexing_tests/test_generate.py` to the scope of tests in GitHub actions?(adding it to `.github/workflows/conda-package.yml`)